src/validation.rs

```diff
@@ -25,6 +25,8 @@ use crate::errors::{new_error, ErrorKind, Result};
 /// // or issuer
 /// validation.set_issuer(&["Me"]); // a single string
 /// validation.set_issuer(&["Me", "You"]); // array of strings
+/// validation.validate_against_system_time(Some(1687425589)) // validate expiry and
```

Comment:
That's a niche function, let's not add it to the doc of the struct that would be confusing